apps/src/calc/CalcVisualizationColumn.jsx

```diff
@@ -0,0 +1,27 @@
+var GameButtons = require('../templates/GameButtons');
+var BelowVisualization = require('../templates/BelowVisualization');
+var ProtectedStatefulDiv = require('../templates/ProtectedStatefulDiv');
+
+var CalcVisualizationColumn = function (props) {
+  return (
+    <span>
+      <ProtectedStatefulDiv id="visualization">
+        <svg version="1.1" id="svgCalc">
+          <image id="background" height="400" width="400" x="0" y="0" xlinkHref="/blockly/media/skins/calc/background.png"></image>
```

Comment:
My approach has been to be less strict about breaking tags onto multiple lines when I have HTML tags (vs. React components). Not sure if that's a good approach.